src/app/zeko/da-layer/scripts/postBatch.ts

```diff
@@ -11,27 +11,35 @@ import { fieldToHex } from "../utils/mina";
 
```

Comment:
How come this file still exists?

Reply:
It's useful for the local testing.

Reply:
I don't see the point of keeping both the Hardhat stuff and the Rust stuff. Keep one.